update style guide to suggest the use of folding block scalar yaml syntax for alt text tags (#3857)
the-turing-way/the-turing-way: book/website/community-handbook/style/style-figures.md

Please make sure that the link to the source is the {term}`digital object identifier <Digital Object Identifier>` not the Zenodo record.
Also ensure that you have created a link to the source using markdown link formatting: `[text](url)`.

Figure information with the block delimited by `---` makes use of [yaml](https://yaml.org/spec/1.2.2/) syntax.

> So you may need to escape certain special characters, this comes up quite often in longer blocks of text,
> such as the alt text tag.

Reviewer: ```suggestion
Some characters (such as ':') have special meaining in YAML and using they may create unexpected results.
This comes up quite often in longer blocks of text, such as the alt text tag.
```

> Using the syntax demonstrated above `alt: >` means you can have multi-line outputs and only need to 

Reviewer: ```suggestion
Using the syntax demonstrated above, `alt: >`, means you can have multi-line outputs and only need to 
```

> escape `|` and `>` like so: `\|` `\>` to get those literal characters in the final output.

Reviewer: ```suggestion
escape `|` and `>` to get those literal characters in the final output.
You escape a character by prepending it with a backslash for example, `\|` and `\>`.
```

> for more on multi-line strings in yaml see: [Demystifying YAML Multiline Strings: An In-Depth Guide](https://thelinuxcode.com/yaml-multiline-strings/) 

Reviewer: ```suggestion
for more on multi-line strings in yaml see: [Demystifying YAML Multiline Strings: An In-Depth Guide](https://thelinuxcode.com/yaml-multiline-strings/).
```

> 
> 

Reviewer: ```suggestion

```
